aiidalab_widgets_base/dicts.py

```diff
@@ -221,3 +221,21 @@
     "Hs": 1.75,
     "Mt": 1.75,
 }
+
+RGB_colors = {
```

Comment:
The value is used as a tuple by converting from a list, so why not define the value as tuple directly?

Reply:
Thanks, done in https://github.com/aiidalab/aiidalab-widgets-base/pull/535/commits/011ec6e180875d7aadd2f90e06fd151a79a8399e. I also renamed RGB_colors to RGB_COLORS, since it is a constant.